src/interface/interfaceProductCategory.ts

```diff
@@ -0,0 +1,20 @@
+interface Category {
+    id: number;
+    name: string;
+    image: string;
+    creationAt: string;
+    updatedAt: string;
+}
+
+interface Product {
+    id: number;
+    title: string;
+    price: number;
+    description: string;
+    images: string[];
+    creationAt: string;
+    updatedAt: string;
+    category: Category;
+}
+
+export default Product;
```

Comment:
do not recommend u use default exports in your project at all, in smaller project ity is aceptable, but in a bigger one, u will get a mess if you will use default exportsd
https://dev.to/phuocng/avoid-using-default-exports-a1c#:~:text=Default%20exports%20can%20be%20imported,and%20using%20different%20import%20names.